### ISSUE TITLE
Can't register for developer account at RightMesh

### ISSUE THREAD

**Issue by tendaimaswaya** [RightMesh/HelloMesh#12]
Failing to register for an account at https://developer.rightmesh.io/ therefore ultimately, i cant use this library

**Comment by Vegen** [comment]
Me too,cannot jump to the registration page.**help!**

**Comment by tendaimaswaya** [comment]
I doubt you'll get a response soon mate. After 4 emails, a tweet and this thread for help, i got nothing

**Comment by Vegen** [comment]
@tendaimaswaya Do you have any alternatives to mesh chat on other platforms?

**Comment by tendaimaswaya** [comment]
Definitely, [Underdark](https://github.com/udark) is a good alternative. Try it

**Comment by Vegen** [comment]
@tendaimaswaya Unfortunately, I can't open their website.([http://underdark.io](http://underdark.io))

**Comment by tendaimaswaya** [comment]
For Android use [this](https://github.com/udark/underdark-android) and follow the documentation there

**Comment by Vegen** [comment]
@tendaimaswaya Sincerely hope to be your friend， What chat software do you usually use in your country? I come from China.

**Comment by Vegen** [comment]
Can it(Underdark) conduct mesh instant chat without network?And is it completely free?

**Comment by tendaimaswaya** [comment]
@Vegen sure, that would be great. I use Whatsapp or Skype, im also comfortable with WeChat.
Yes, underdark can conduct mesh instant chat without network and is absolutely free

**Comment by Vegen** [comment]
@tendaimaswaya I'm looking forward to your adding me as a friend in wechat,and my wechat account number is DavidHuweijian

**Comment by Vegen** [comment]
@tendaimaswaya This library is not very easy to use, found that the number of devices is not synchronized, offline devices can not be detected in time.If you know there are other solutions, look forward to your suggestions. Thank you.

**Comment by azizimusa** [comment]
i cant find any mesh network project as good as Underdark. I use it on my android project and i liked it. I just have problem with the iOS project. Can run on phone but _abort with payload appear. If anyone know how to fix it. Please tell me.

[[Update]]
need to add Privacy - Bluetooth Always Usage Description in xcode. Then It can run fine.

**Comment by ototech7** [comment]
hello azizimusa, that's really helpful for everyone who might get stuck on same iOS issue. you are correct, from iOS 13 apple made it required to add Privacy, Bluetooth Always Usage Description inside Info.plist when using bluetooth in background. otherwise app will crash with that "_abort with payload" message.

if it still not working after you added it, just double check the permission keys like NSBluetoothPeripheralUsageDescription and also maybe NSLocationAlwaysAndWhenInUseUsageDescription if your app does discovery while background. also make sure you tick the "Uses Bluetooth LE accessories" in Background Modes under Capabilities.
